doc/create_ramp_event.rst

```diff
@@ -53,7 +53,27 @@ This config file should look like::
       more precisely the :ref:`conda workers <conda_env_worker>` to have more
       information.
 
-Finally, you can easily deploy the event (adding both problem and event to the
+Before you continue make sure that::
+    1.  you have the private and the public data in the following directory
+        structure::
+
+        ├──ramp_deployment
+            ├── ramp-kits
+                ├── iris
+                    ├── data  # your public data
+                        ├── test
+                        ├── train
+            ├── ramp-data
+                ├── iris
+                    ├── data  # your private data
+                        ├── test
+                        ├── train
+
```

Comment:
Maybe use the ` └──` notation for when the last file/dir

Reply:
Also the file dir structure could be more clear with more lines, for example: https://paris-saclay-cds.github.io/ramp-docs/ramp-workflow/dev/workflow.html#overall-directory-structure

Reply:
Actually this exact directory structure is not strictly necessary. This is why we changed to copying the data files from `ramp-data` to `ramp-kits` - to be robust to differences in directory structure. We suggest this structure in the `Deploy a specific RAMP event` section above with the commands given, but this exact structure is not required. It might be worthwhile to mention this.
Also for files add a `/` at the end, to indicate that they are files.
How the data files look can also vary. For the human locomotion challenge, the data were in `test` and `train` directories and the `get_train_data` and `get_test_data` were written to load files from a folder. For other challenges, it is common that the test and train data are just files within `data`. It would be good to make it clear that how the data files look can vary.